dev/environment-dev.yml

```diff
@@ -39,7 +39,6 @@ dependencies:
   - securesystemslib
   # libmambapy build dependencies
   - scikit-build
-  - build
```

Comment:
Why add this?

Reply:
[The `build` package on `conda-forge`](https://prefix.dev/channels/conda-forge/packages/build) has been yanked.

Reply:
Isn't it superseded by something else?

Reply:
It is superseded by `python-build` (see https://github.com/conda-forge/conda-forge.github.io/issues/2269), but `python-build` itself is not required it seems.